Content.Shared/Standing/StandingStateSystem.cs

```diff
@@ -63,14 +59,32 @@ public bool IsDown(EntityUid uid, StandingStateComponent? standingState = null)
             _appearance.SetData(uid, RotationVisuals.RotationState, RotationState.Horizontal, appearance);
 
             // Change collision masks to allow going under certain entities like flaps and tables
+            // Also change layers so projectiles will pass over laying down entities.
             if (TryComp(uid, out FixturesComponent? fixtureComponent))
             {
                 foreach (var (key, fixture) in fixtureComponent.Fixtures)
                 {
+                    if (fixture.CollisionLayer == (int) CollisionGroup.MobLayer && dropHeldItems)
+                    {
+                        standingState.StandingCollisionLayers.TryAdd(key, fixture.CollisionLayer);
+                        standingState.LayerChangedFixtures.Add(key);
+
+                        _physics.SetCollisionLayer(uid, key, fixture,
+                            (int) CollisionGroup.LayingDownMobLayer,
+                            manager: fixtureComponent);
+                    }
+                    //Returns CollisionLayer to the original one if laid down on something while already being down.
+                    else if (fixture.CollisionLayer == (int) CollisionGroup.LayingDownMobLayer && !dropHeldItems)
+                    {
+                        _physics.SetCollisionLayer(uid, key, fixture,
+                            standingState.StandingCollisionLayers[key],
+                            manager: fixtureComponent);
+                    }
+
                     if ((fixture.CollisionMask & StandingCollisionLayer) == 0)
```

Comment:
As above doing exact bitmask checks like this is a bad idea, especially when you're trying to infer mob functionality from it.

Just add whatever collision layer is needed to align with the target's mask.

Reply:
I now add/subtract layers /masks when needed